test/unit/auth/auth-config.spec.ts

```diff
@@ -727,20 +727,34 @@ describe('OIDCConfig', () => {
     issuer: 'https://oidc.com/issuer',
     displayName: 'oidcProviderName',
     enabled: true,
+    clientSecret: 'CLIENT_SECRET',
```

Comment:
It may be a good idea to add one test in `auth-api-request.spec.ts` for code flow:
https://github.com/firebase/firebase-admin-node/blob/011c53043896587e2d41228b3c8fdc463faff56a/test/unit/auth/auth-api-request.spec.ts#L3496